Cooking and Drinks Supplemental Update

Unknownity opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## About The Pull Request

This PR adds in additional features for the Mess Technician to mess with and a general "quality of life" upgrades of the current state of cooking and making drinks.
Soda and Booze dispensers have been fixed to be able to wrenchable now (a previous PR in gitlab era tried making them wrenchable but forgot a crucial detail), with one exception currently being the soda dispenser in the Almayer main kitchen (not officer's mess) since its bolted to the wall.

The soda dispenser has been upgraded to include additional drinks, these being tomato,carrot,berry,grape,lemon and banana juices. Ice Tea has been removed from the dispenser since it is an easily made mixed drink (tea+ice). The secret mode of the soda dispenser has been adjusted to keep up with the theme of being a non-alcoholic dispenser (previously it gave you an alcoholic drink and an entire can of grape soda), If you want to spoil yourself on what it dispenses now, check the code.

The booze dispenser has been adjusted with what reagents it dispenses, all the non alcoholic liquids except for very basic ones (water,sodawater,sugar,ice) have been removed as you can find them in the soda dispenser and it gained two new drinks called thirteenloko which has been previously the secret drink of the soda dispenser and tequila. The booze dispenser has obtained an additional secret drink.

The groceries variant of the fridge has obtained new ingredients inside, these being: cherries, limes, lemons, oranges and bananas. So the Mess Technician isn't forced to grow these at the start when they could very easily exist in the fridge.

The Officer's Mess meat fridge has gained some carp fillets instead of already made tofu products, Officers should obviously have some exotic meat right?

A new condiment has surfaced, it is a cold sauce bottle that can be found in Shivas Snowball (it's made from there) and in the USS Almayer's main kitchen freezer crate. the sprite is simply a resprite of one of the hot sauce bottles.

some drinks lacked sprites (were broken) such as Ale and Tequila Sunrise, probably weren't the only ones but so far only these two were found and fixed. Ale's got its bottled sprite back thanks to Bay12
fixes #1733 

A new alcoholic drink was added, the Mojito. we already had an unused sprite in the repo for it, so I just added it fully in the game.

someone put the wrong id for universal enzyme in the primordial cheese, so fixed that.

## Why It's Good For The Game

Provides additional options for the Mess Technician to play with, removes the necessity to grow very common fruit right at the start just because there's lack of any in the fridge. Makes Mess Technicians also able to bring their soda/booze dispensers from the Colony Bars or Almayer's Officer Mess down to the FOB

## Changelog

<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl: Unknownity
add: The Soda and Booze dispensers have obtained additional drinks.
add: The grocery fridge has been filled with more unique fruit and vegetables.
add: The Officer's Mess meat fridge has been filled with carp meat.
add: A new condiment bottle has surfaced from Shivas Snowball, you can find it there and on the USS Almayer inside the main kitchen's freezer crate.
add: Added a new alcoholic drink, the Mojito! The recipe to make it is roughly the same as in real life.
fix: Soda and Booze dispensers are now actually wrenchable.
fix: Fixed certain drinks not having any sprites.
fix: Fixed Primordial Cheese not containing universal enzyme.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->

stanalbatross commented on code/modules/reagents/chemistry_machinery/chem_dispenser.dm:
```diff
@@ -239,12 +254,26 @@
+		if(do_after(user, 20, INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD))
+			if(!src) return
+			playsound(src.loc, 'sound/items/Ratchet.ogg', 25, 1)
+			switch (anchored)
+				if (0)
+					anchored = 1
+					user.visible_message("[user] tightens the bolts securing \the [src] to the surface.", "You tighten the bolts securing \the [src] to the surface.")
+				if (1)
+					user.visible_message("[user] unfastens the bolts securing \the [src] to the surface.", "You unfasten the bolts securing \the [src] to the surface.")
+					anchored = 0
```
TRUE FALSE please, also can't you just call parent?

stanalbatross commented on code/modules/reagents/chemistry_machinery/chem_dispenser.dm:
```diff
@@ -202,23 +202,38 @@
+			if(!src) return
+			playsound(src.loc, 'sound/items/Ratchet.ogg', 25, 1)
+			switch (anchored)
+				if (0)
+					anchored = 1
+					user.visible_message("[user] tightens the bolts securing \the [src] to the surface.", "You tighten the bolts securing \the [src] to the surface.")
+				if (1)
+					user.visible_message("[user] unfastens the bolts securing \the [src] to the surface.", "You unfasten the bolts securing \the [src] to the surface.")
+					anchored = 0
```
same

stanalbatross reviewed (changes requested):
the rest is fine

kugamo reviewed (approved):
Sprites are fine.

moltoretardo commented:
Add that it fixes #1733

Unknownity commented:
> Add that it fixes #1733
In the about: "some drinks lacked sprites (were broken) such as Ale and Tequila Sunrise, probably weren't the only ones but so far only these two were found and fixed. Ale's got its bottled sprite back thanks to Bay12"

and changelog "fix: Fixed certain drinks not having any sprites." as it contains multiple.
